recipes/libxml2/all/conanfile.py

```diff
@@ -234,6 +248,8 @@ def package_info(self):
         self.cpp_info.includedirs.append(os.path.join("include", "libxml2"))
         if not self.options.shared:
             self.cpp_info.defines = ["LIBXML_STATIC"]
+        if self.options.include_utils:
+            self.env_info.path.append(os.path.join(self.package_folder, "bin"))
```

Comment:
```suggestion
        bindir = os.path.join(self.package_folder, "bin")
        self.output.info("Appending PATH environment variable: {}".format(bindir))
        self.env_info.PATH.append(bindir)
```

This is our standard three-liner for adding to the PATH across recipes

Reply:
Is there a guideline that I can take a look to learn this coding conventions (and adopt them also in my recipes)?

Reply:
At the moment not really, but we are working on better docs! :)